packages/cuba-react/src/app/Security.ts

```diff
@@ -24,17 +29,27 @@ export class Security {
   };
 
   @action
-  loadPermissions() {
+  async loadPermissions() {
+
+    await this.cubaREST.refreshApiVersion();
```

Comment:
I think we should use `#isFeatureSupported` instead of explicit call to refresh api version.